test/unit/throw.spec.js

```diff
@@ -91,7 +91,7 @@ describe('a test that throws', function () {
 
     it('should not pass if throwing async and test is async', function (done) {
       var test = new Test('im async and throw null async', function (done2) {
-        process.nexTick(function () {
+        process.nextTick(function () {
           throw null;
         });
       });
```

Comment:
Judging from the looks of these corrected tests, I *think* they were *supposed* to `throw undefined` and `throw null` in the `process.nextTick` callback, *but* were actually throwing due to `process.nexTick` being undefined, and the tests only checked that the attempt failed somewhere along the line and not that it got to the callback and failed by throwing out of the callback. I don't know off the top of my head whether those tests should be checking something to be sure that `undefined` and `null` are what's thrown respectively, but they should probably be doing something to prevent treating throws outside the callback as passing. Perhaps wrapping the synchronous portion  with `try {` and `} catch (ignore) { /* ensure that early failure due to any problems reaching the callback is not mistaken for the failure we're expecting to get out of the callback when it is asynchronously called */ }`?

In any case, some extra eyes on them would be helpful just to make sure that we really are making them better... I'm not absolutely certain I've ascertained their purpose correctly here, and, frankly, I can't quite figure out why there are tests failing in CI now...

Reply:
I'm going to split this out and leave it for someone else to solve.
I'm on a spelling binge. While I do understand JS and node, I don't really want to spend the time figuring out the right magic to make the test happy.